django/contrib/contenttypes/admin.py

```diff
@@ -98,7 +98,7 @@ def get_formset(self, request, obj=None, **kwargs):
             exclude = []
         else:
             exclude = list(self.exclude)
-        exclude.extend(self.get_readonly_fields(request, obj))
+        exclude.extend(self.get_uneditable_fields(request, obj))
```

Comment:
Ah. Ignore this - GitHub had hidden the diff of `django/contrib/admin/options.py` when I skimmed through last time.